a11ym

```diff
@@ -41,6 +42,11 @@ program
         __dirname + '/a11ym_output'
     )
     .option(
+        '-p, --concurrency <concurrency>',
```

Comment:
`p` ?  even if it probably stands for parallels, I would say it's not common

Maybe `workers` and `w` would be more easily recognizable ?